utils.js

```diff
@@ -696,6 +696,15 @@ function getType(obj) {
   return Object.prototype.toString.call(obj).slice(8, -1);
 }
 
+function formatProxyPresence(presence, userID) {
+  if(presence.lat === undefined) return null;
+  return {
```

Comment:
Can we add a `type: "presence" to this to make it match the signature of the other event?